packages/adapter/addon/rest.js

```diff
@@ -1093,14 +1095,19 @@ const RESTAdapter = Adapter.extend(BuildURLMixin, {
       options.headers = {};
     }
 
-    if (options.data && options.type !== 'GET') {
-      let contentType = options.contentType || 'application/json; charset=utf-8';
-      options.headers['content-type'] = contentType;
-    }
+    let contentType = options.contentType || this.defaultContentType;
 
     if (get(this, 'useFetch')) {
+      if (options.data && options.type !== 'GET') {
+        if (!options.headers['Content-Type'] && !options.headers['content-type']) {
+          options.headers['content-type'] = contentType;
+        }
+      }
       options = fetchOptions(options, this);
     } else {
+      if (options.data && options.type !== 'GET') {
```

Comment:
A GET request with a non-simple content-type will result in a preflight request.  One consequence of adding the `Content-Type` header on GET (where we used to not) is the backend server perhaps doesn't expect this and won't return the right a-c-a-o header.

Reply:
Can we leave this explanation of this conditional as a comment in the code?